codeguru/views.py

```diff
@@ -116,6 +116,11 @@ def new_center(request):
             new_center = Center(name=name, ticker=ticker)
             new_center.save()
             return redirect("group")
+        try:
+            validate_center(form.name)
+            return error(request, form.errors)
+        except:
+            return error(request, gettext("Center should be represented with 3 letters in english."))

```

Comment:
Move the `gettext` to `validate_center`, that way the error message will be included in `form.errors`. Alternatively, use `gettext` on each error in `form.errors` (I like this option less though)

This will make this awkward check redundant, and save us having to write out the error message twice.

Reply:
The problem with just displaying form.errors is that is also shows the error's path.
The path is not translated into hebrew.
I cannot find a way to prevent it from showing the path.
![image](https://github.com/user-attachments/assets/a34355f0-67da-42cf-80ee-3389184420cd)

Reply:
Resolved by changing `errors.html`
![image](https://github.com/user-attachments/assets/796ab78f-238e-449e-aba3-695344973cd2)